src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/CodeSample.razor.cs

```diff
@@ -6,19 +6,22 @@ namespace Bit.Client.Web.BlazorUI.Playground.Web.Components
 {
     public partial class CodeSample
     {
-        private ElementReference CodeReference { get; set; }
-        private bool ShowCode { get; set; }
+        private bool showCode { get; set; }
+        private string copyButtonTitle { get; set; } = "copy";
```

Comment:
what's the use of this private property? shouldn't it be used as a simple string in the razor file?